extra/release_tool.py

```diff
@@ -721,6 +721,14 @@ def generate_new_tags(state, tag_avail, final):
         if not next_tag_avail[repo.git]['already_released'] and final:
             next_tag_avail[repo.git]['already_released'] = True
 
+    # Make the integration version available to the UI through an environment variable
+    # and store
+    for repo in REPOS.values():
+        if repo.git == "integration":
+            os.environ('INTEGRATION_VERSION', next_tag_avail[repo.git]['build_tag'])
```

Comment:
Any point of having this line?

Reply:
Acceptance criteria

- [x] During release the integration version is stored in a file (e.g. json)
- [x] This file or its contents is mounted in a place accessible to the UI or made available to the UI *through environment variables*